src/DependencyInjection/Configuration.php

```diff
@@ -23,6 +23,11 @@ public function getConfigTreeBuilder(): TreeBuilder
 					->isRequired()
 					->cannotBeEmpty()
 				->end()
+				->scalarNode('auth_token')
+					->info('Unleash auth token')
+					->isRequired()
```

Comment:
The `auth_token` option must not be required, because for some installations (eg. Gitlab Feature flags) it does not require an `Authorization` header